frontend/javascripts/oxalis/store.js

```diff
@@ -101,6 +101,22 @@ export type BoundingBoxObject = {
   +depth: number,
 };
 
+export type UserBoundingBoxToServer = {
```

Comment:
I'm confused what the `To` refers to. Would `ServerUserBoundingBox` be more in line with our other type names?

Reply:
Indeed the naming is n't optimal. The problem is that the server sends bounding boxes as a certain type but receives them a different type. This inconsistency shall be tackled in another PR.

`ServerUserBoundingBox` is already in use for the type that is sent **by** the server. Thus I named the type that is sent **to** the server `UserBoundingBoxToServer`.
Suggestions for a better name are welcome :smile:

Reply:
:see_no_evil: I see, that makes sense. You could rename the `ServerUserBoundingBox` to `UserBoundingBoxFromServer`, but I have no strict opinion on this :)

Reply:
I like your idea. :+1: